README.md

```diff
@@ -152,6 +152,21 @@ stripe.customers
       });
   });
 ```
+### Usage with Deno
+
+As of 11.16.0, stripe-node provides a `deno` export target. You may import stripe-node in Deno through npm specifiers or a third party CDN like esm.sh.
+
+Import using npm specifiers:
+```js
+import Stripe from 'npm:stripe';
+```
+
+Import using esm.sh:
```

Comment:
I think we should encourage `npm:stripe` as a safer alternative to having library re-processed via a third party service.

Reply:
I removed the references to esm.sh, if a user really wants to use a CDN they can refer to the Deno docs